internal/boxcli/add.go

```diff
@@ -37,7 +36,7 @@ func addCmd() *cobra.Command {
 				return nil
 			}
 			err := addCmdFunc(cmd, args, flags)
-			if errors.Is(err, nix.ErrPackageNotFound) {
+			if err != nil {
```

Comment:
This will mask any other error (including not logging them to Sentry)

I think we should change https://github.com/jetpack-io/devbox/blob/main/internal/searcher/client.go#L63-L63 to return `ErrPackageNotFound` (maybe move it to searcher pkg) so we only show this error message for that error. 

There's also this potential error as well: https://github.com/jetpack-io/devbox/blob/main/internal/nix/input.go#L260-L260

not sure if we need any special handling for that.

Reply:
Just checked, no need for https://github.com/jetpack-io/devbox/blob/main/internal/nix/input.go#L260-L260 as that only happens when a user does:
```
devbox add nodejs@
```
with the `@` but no version